src/core/grid_based_algorithms/lbgpu.cpp

```diff
@@ -102,7 +102,7 @@ LB_parameters_gpu lbpar_gpu = {
     0.0};
 
 /** this is the array that stores the hydrodynamic fields for the output */
-LB_rho_v_pi_gpu *host_values = nullptr;
+std::vector<LB_rho_v_pi_gpu> host_values(0);
```

Comment:
Here too, why is this a `vector`?

Reply:
There is multiple cals of resize for host_values in lb_interface.cpp :
`host_values.resize(lbpar_gpu.number_of_nodes);`
so the size of host_values is not statically known.

Reply:
Ah sorry, I did not realize that this is different from `host_print_values`